### DISCUSSION TITLE
Guess 🎥🍿 catchphrase

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#991] [Galaxy Trivia] [answered]
> Life is like 📦🍫, you 🤷‍♀️💭 what you're gonna get

**Bonus:** Name the character and actor.

**Comment by peterbaccaro** [comment]
The catchphrase is **Life is like a box of chocolates. You never know what you're gonna get**.

This famous line is spoken by **Forrest Gump**, played by **Tom Hanks** in the movie Forrest Gump (directed by Robert Zemeckis, 1994).

**Comment by thuongtruong1009** [comment, marked as answer]
The catchphrase you're referring to is from the movie Forrest Gump (1994). The complete catchphrase is "Life is like a box of chocolates, you never know what you're gonna get."

The character who says this line is Forrest Gump, portrayed by actor Tom Hanks.
